.editorconfig

```diff
@@ -26,3 +26,52 @@ dotnet_diagnostic.MSML_ParameterLocalVarName.severity = none
 
 # MSML_TypeParamName: Type parameter name not standard
 dotnet_diagnostic.MSML_TypeParamName.severity = none
+
+[test/Microsoft.ML.CodeAnalyzer.Tests/**.cs]
+# BaseTestClass does not apply for analyzer testing.
+# MSML_ExtendBaseTestClass: Test classes should be derived from BaseTestClass
+dotnet_diagnostic.MSML_ExtendBaseTestClass.severity = none
+
+[test/Microsoft.Extensions.ML.Tests/**.cs]
+# MSML_ExtendBaseTestClass: Test classes should be derived from BaseTestClass
+dotnet_diagnostic.MSML_ExtendBaseTestClass.severity = none
```

Comment:
This block disables the `MSML_ExtendBaseTestClass` rule for source files under **test/Microsoft.Extensions.ML.Tests**. This is required because one or more classes in this folder is considered a test class by this analyzer but does not extend `BaseTestClass`. However, this pull request only implements the analyzer; a future pull request will remove these lines from the configuration file and update the test classes to correct the violations.